tests/js/spec/views/dashboardsV2/widgetBuilder/widgetBuilder.spec.tsx

```diff
@@ -1893,11 +1893,11 @@ describe('WidgetBuilder', function () {
       expect(screen.queryByPlaceholderText('Enter Equation')).not.toBeInTheDocument();
 
       await waitFor(() => {
-        expect(eventsStatsMock).toHaveBeenCalledWith(
-          '/organizations/org-slug/events-stats/',
+        expect(eventsv2Mock).toHaveBeenCalledWith(
```

Comment:
This was a test that should've been focusing on a table, so the assertion on `events-stats` was incorrect.